src/store/auth/selectors.ts

```diff
@@ -71,7 +71,8 @@ export const getFilteredUsers = createSelector(
     return users.filter(u => {
       return [u.localId, u.displayName, u.email || '', u.phoneNumber || '']
         .join('\n')
-        .includes(filter);
+        .toLocaleLowerCase()
```

Comment:
Please use `toLocaleUpperCase` instead (and same below) due to l10n concerns: https://stackoverflow.com/a/14128850

(Side note: It's still not perfect, but better than lowering the case. The best way would be using a locale-aware method, but I'm not _aware_ of such a method for `.includes`.)

Reply:
Looks like this only applies if we want to convert it back
I also have 0 preference here, so switched to upperCase